libsrc/apple2/ser/a2.gs.s

```diff
@@ -314,20 +314,19 @@ SER_CLOSE:
 
 SER_OPEN:
         bit     $C082                   ; Check if this is a IIgs
-        lda     $FE1F                   ; https://prodos8.com/docs/technote/misc/07/
-        cmp     #$60                    ; Everything but the IIgs has an RTS there
-        bne     HardwareFound
+        sec
+        jsr     $FE1F                   ; https://prodos8.com/docs/technote/misc/07/
```

Comment:
Please don't use a URL here. URLs get invalid at some point in time. Rather use https://github.com/cc65/cc65/blob/master/libsrc/apple2/get_ostype.s#L11